src/domain/shared/components/withElevationOnHover.tsx

```
@@ -30,6 +30,13 @@ const withElevationOnHover = <El, P extends ComponentProps<El>>(Component: Compo
     );
     const setInitialElevation = useCallback(() => setElevation(initialElevation), [initialElevation]);
 
+    if (componentProps['$$typeof']) {
+      delete componentProps['$$typeof'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)